project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/c4m_features_homepage.module

```diff
@@ -122,7 +123,6 @@ function c4m_features_homepage_voices_views_carousel() {
     ->fieldCondition('c4m_banner', 'fid', '', '!=')
     ->fieldCondition('c4m_intro_text', 'value', '', '!=')
     ->propertyOrderBy('sticky', 'DESC')
-    ->propertyOrderBy('promote', 'DESC')
```

Comment:
@soniCaH 
Please confirm that removing this line is part of the request.
> Desired effect:
show items with a banner image (as it is now)
most recently created sticky items first
then recently created (descending creation date order)
Apply an image style instead of outputting the entire image